clippy.toml

```diff
@@ -1,10 +1,15 @@
+# TODO: If possible, gate this so that #[cfg(miri)] is mostly only contained within oxdock-fs
+# and oxdock-process crates, where the other crates are agnostic to the test runner.
+
+# TODO: Update this description to cover "types" and "processes"
```

Comment:
Be sure to handle this.

Reply:
Updated via: https://github.com/jzombie/oxdock-rs/pull/16/commits/33ac1862f83f8ae8ba64c878e5e7fd5463a47335